src/test/resources/org/jenkinsci/plugins/ewm/casc/configuration-as-code.yaml

```diff
@@ -11,6 +11,4 @@ unclassified:
           - diskId: "disk1"
             displayName: "disk one display name"
             masterMountPoint: "/tmp"
-        displayName: "diskpool1 display name"
-
-
+        displayName: "diskpool1 display name"
```

Comment:
Sorry to bother you with these blank lines, it's really not a big deal, but at the **end** of the file it's ok to have **one** blank line :)
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file
Same for the next file: obsolete-configuration-as-code.yaml